src/api/app/views/webui/package/show.html.erb

```diff
@@ -38,6 +38,13 @@
               at <%= link_to(elide(dpackage.project.name, 44), :action => :show, :controller => :package, :project => dpackage.project.name, :package => dpackage.name) %>
             </li>
         <% end %>
+        <% if @package.project != @project %>
+            <li>
+              <%= sprite_tag 'information', title: '' %>
+              Sources life in linked project
```

Comment:
`s/life/live/`

I would like this more:

`Sources linked from project $project`